Flagsmith.FlagsmithClient/FlagsmithClient.cs

```diff
@@ -95,6 +103,13 @@ public class FlagsmithClient : IFlagsmithClient
                 _pollingManager = new PollingManager(GetAndUpdateEnvironmentFromApi, EnvironmentRefreshIntervalSeconds);
                 _ = _pollingManager.StartPoll();
             }
+            if (CacheConfig.Enabled)
+            {
+                _regularFlagListCache = new RegularFlagListCache(new DateTimeProvider(),
+                    GetEnvironmentFlags().Result,
+                    CacheConfig.DurationInMinutes);
+                _flagListCacheDictionary = new Dictionary<string, IdentityFlagListCache>();
+            }
         }
 
         /// <summary>
```

Comment:
I think we should also add  CacheConfig in IFlagsmithConfiguration and in this constructor.

Reply:
rows 120-123

Reply:
Added!!